backend/app/api/endpoints/adapter/tasks.py

```
@@ -338,6 +339,36 @@ def delete_archived_tasks(
     return {"message": "Archived chats deleted successfully", "count": count}
 
 
+@router.delete("/bulk", response_model=TaskArchiveBatchResponse)
+def bulk_delete_tasks(
+    request: TaskBulkDeleteRequest,
+    client_origin: ClientOriginQuery = CLIENT_ORIGIN_FRONTEND,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)